WebHostLib/upload.py

```diff
@@ -156,11 +158,11 @@ def uploads():
                     # noinspection PyBroadException
                     try:
                         multidata = file.read()
-                        MultiServer.Context.decompress(multidata)
+                        slots, multidata = parse_multidata(multidata)
                     except Exception as e:
                         flash(f"Could not load multidata. File may be corrupted or incompatible. ({e})")
                     else:
-                        seed = Seed(multidata=multidata, owner=session["_id"])
+                        seed = Seed(multidata=multidata, slots=slots,owner=session["_id"])
```

Comment:
Curious. This this ever work without assigning slots?

Also
```suggestion
                        seed = Seed(multidata=multidata, slots=slots, owner=session["_id"])
```